Integrations/integration-SumoLogic.yml

```diff
@@ -261,7 +261,8 @@ script:
             }
             var headers = 'headers' in args ? argToList(args.headers) : undefined;
             var s = search(query, args.from, args.to, a2i(args.limit, defaultLimit), a2i(args.offset, 0), args.timezone,
-                a2i(args.maxTimeToWaitForResults, defaultSearchTimeout) * 60, args.byReceiptTime, a2i(params.sleepBetweenChecks, defaultSleep));
+                a2i(args.maxTimeToWaitForResults, defaultSearchTimeout) * 60, args.byReceiptTime, a2i(params.sleepBetweenChecks, defaultSleep),
+                params.fetchRecords);
```

Comment:
I think in this case we better add new argument to `search` command, `waitForSearchComplete` with default value false, so user will have the flexibility on a command level decide if he wants to wait or not.
params.fetchRecords usually we refer to `fetch-incidents`

Reply:
I’m happy to implement this, but I think this lays a trap for unsuspecting users. The reason we have discovered the flaw was that our indicator enhancement did not return any values, while the Sumo search ran manually returned it correctly.
The “terminate shortly” logic is flawed when you want to have accurate aggregate results: the API response returns the aggregations correctly, but over incomplete data. So whoever sets `fetchAggregations` without `waitForSearchComplete=true` ends up receiving incomplete results